### ISSUE TITLE
1.10.1: pytest is failing in `tests/test_postgresql.py::PostgresqlAutogenRenderTest::test_inline_exclude_constraint_text` unit

### ISSUE THREAD

**Issue by kloczek** [sqlalchemy/alembic#1197]
I'm packaging your module as an rpm package so I'm using the typical PEP517 based build, install and test cycle used on building packages from non-root account.
- `python3 -sBm build -w --no-isolation`
- because I'm calling `build` with `--no-isolation` I'm using during all processes only locally installed modules
- install .whl file in </install/prefix>
- run pytest with $PYTHONPATH pointing to sitearch and sitelib inside </install/prefix>
- build is performed in env which is *`cut off from access to the public network`* (pytest is executed with `-m "not network"`)

Here is pytest output:
<details>

```console
+ PYTHONPATH=/home/tkloczko/rpmbuild/BUILDROOT/python-alembic-1.10.1-2.fc35.x86_64/usr/lib64/python3.8/site-packages:/home/tkloczko/rpmbuild/BUILDROOT/python-alembic-1.10.1-2.fc35.x86_64/usr/lib/python3.8/site-packages
+ /usr/bin/pytest -ra -m 'not network' -q
============================= test session starts ==============================
platform linux -- Python 3.8.16, pytest-7.2.2, pluggy-1.0.0
rootdir: /home/tkloczko/rpmbuild/BUILD/alembic-rel_1_10_1, configfile: setup.cfg
collected 1749 items

tests/test_autogen_composition.py .ss.....                               [  0%]
tests/test_autogen_diffs.py sssssssssss...sss.s......s.............sss.. [  2%]
.......s....................sssss........ss......sssss.................. [  7%]
............................s.                                           [  8%]
tests/test_autogen_indexes.py .......................................... [ 11%]
..........s...........................................s................. [ 15%]
...s..s....................s.                                            [ 16%]
tests/test_autogen_render.py ........................................... [ 19%]
........................................................................ [ 23%]
..............................                                           [ 25%]
tests/test_batch.py .................................................... [ 28%]
......ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 32%]
sssssssssssssssssssssssssssssssssssssss.....s.ss.......s....ss.......... [ 36%]
....s...........................                                         [ 38%]
tests/test_bulk_insert.py ..................                             [ 39%]
tests/test_command.py .................................................. [ 42%]
.......................................                                  [ 44%]
tests/test_config.py ..........................                          [ 45%]
tests/test_editor.py ..........                                          [ 46%]
tests/test_environment.py .............                                  [ 47%]
tests/test_external_dialect.py ...                                       [ 47%]
tests/test_impl.py ....                                                  [ 47%]
tests/test_mssql.py .....ssss.........sss............................... [ 50%]
......ss                                                                 [ 51%]
tests/test_mysql.py sssssssssssssss..................................... [ 54%]
............                                                             [ 54%]
tests/test_offline_environment.py ..........................             [ 56%]
tests/test_op.py ....................................................... [ 59%]
...................................s..................s..............    [ 63%]
tests/test_op_naming_convention.py .............                         [ 64%]
tests/test_oracle.py .....ssssss.sssss..................sssss.s....s     [ 66%]
tests/test_post_write.py .............                                   [ 67%]
tests/test_postgresql.py ss....sssssss.......F......ssssssssssssssssssss [ 70%]
sssssssssss.sss.sss.........sssss..............s..                       [ 73%]
tests/test_revision.py ................................................. [ 75%]
........................................................................ [ 79%]
.........                                                                [ 80%]
tests/test_script_consumption.py ...ssss.................s.............. [ 82%]
...............................                                          [ 84%]
tests/test_script_production.py .......................................  [ 86%]
tests/test_sqlite.py .....................s.s..                          [ 88%]
tests/test_stubs.py ss                                                   [ 88%]
tests/test_suite.py ssssssssssssssss........s.sssssss...........sss..... [ 91%]
......sssssssssssss.........sssssssssssss..............                  [ 94%]
tests/test_version_table.py ............................                 [ 95%]
tests/test_version_traversal.py ........................................ [ 98%]
..............................                                           [100%]

=================================== FAILURES ===================================
_______ PostgresqlAutogenRenderTest.test_inline_exclude_constraint_text ________
Traceback (most recent call last):
  File "/home/tkloczko/rpmbuild/BUILD/alembic-rel_1_10_1/tests/test_postgresql.py", line 1210, in test_inline_exclude_constraint_text
    autogenerate.render_op_text(autogen_context, op_obj),
  File "/home/tkloczko/rpmbuild/BUILD/alembic-rel_1_10_1/alembic/autogenerate/render.py", line 132, in render_op_text
    return "\n".join(render_op(autogen_context, op))
  File "/home/tkloczko/rpmbuild/BUILD/alembic-rel_1_10_1/alembic/autogenerate/render.py", line 125, in render_op
    lines = util.to_list(renderer(autogen_context, op))
  File "/home/tkloczko/rpmbuild/BUILD/alembic-rel_1_10_1/alembic/autogenerate/render.py", line 214, in _add_table
    table = op.to_table()
  File "/home/tkloczko/rpmbuild/BUILD/alembic-rel_1_10_1/alembic/operations/ops.py", line 1159, in to_table
    return schema_obj.table(
  File "/home/tkloczko/rpmbuild/BUILD/alembic-rel_1_10_1/alembic/operations/schemaobj.py", line 219, in table
    constraints = [
  File "/home/tkloczko/rpmbuild/BUILD/alembic-rel_1_10_1/alembic/operations/schemaobj.py", line 220, in <listcomp>
    sqla_compat._copy(elem, target_table=t)
  File "/home/tkloczko/rpmbuild/BUILD/alembic-rel_1_10_1/alembic/util/sqla_compat.py", line 214, in _copy
    return schema_item._copy(**kw)  # type: ignore[union-attr]
  File "/usr/lib64/python3.8/site-packages/sqlalchemy/dialects/postgresql/ext.py", line 273, in _copy
    c = self.__class__(
  File "/usr/lib64/python3.8/site-packages/sqlalchemy/dialects/postgresql/ext.py", line 216, in __init__
    expressions, operators = zip(*elements)
ValueError: not enough values to unpack (expected 2, got 0)
=========================== short test summary info ============================
SKIPPED [2] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_autogen_composition.AutogenerateDiffTestWSchema' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [7] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_autogen_diffs.AutogenCrossSchemaTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [2] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_autogen_diffs.AutogenCustomVersionTableAndSchemaTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [2] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_autogen_diffs.AutogenCustomVersionTableSchemaTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [3] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_autogen_diffs.AutogenDefaultSchemaTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_autogen_diffs.AutogenSystemColTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_autogen_diffs.AutogenerateDiffTestWSchema' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareMetadataToInspectorTest_sqlite+pysqlite_3_41_0::test_datetime_comparisons[cola0-colb0-False-_exclusions_00] (call)' : not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareMetadataToInspectorTest_sqlite+pysqlite_3_41_0::test_datetime_comparisons[cola1-colb1-True-_exclusions_01] (call)' : not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareMetadataToInspectorTest_sqlite+pysqlite_3_41_0::test_datetime_comparisons[cola2-colb2-True-_exclusions_02] (call)' : not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareMetadataToInspectorTest_sqlite+pysqlite_3_41_0::test_introspected_columns_match_metadata_columns[cola17-_exclusions_017] (call)' : not mysql and not mariadb
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareMetadataToInspectorTest_sqlite+pysqlite_3_41_0::test_string_comparisons[cola10-colb10-False-_exclusions_010] (call)' : not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareMetadataToInspectorTest_sqlite+pysqlite_3_41_0::test_string_comparisons[cola11-colb11-False-_exclusions_011] (call)' : not mysql and not mariadb
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareMetadataToInspectorTest_sqlite+pysqlite_3_41_0::test_string_comparisons[cola12-colb12-True-_exclusions_012] (call)' : not mysql and not mariadb
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareMetadataToInspectorTest_sqlite+pysqlite_3_41_0::test_string_comparisons[cola13-colb13-False-_exclusions_013] (call)' : not mysql and not mariadb
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareMetadataToInspectorTest_sqlite+pysqlite_3_41_0::test_string_comparisons[cola14-colb14-True-_exclusions_014] (call)' : not mysql and not mariadb
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareServerDefaultTest_sqlite+pysqlite_3_41_0::test_server_default_no_false_positives[DATETIME-(getdate())] (call)' : not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareServerDefaultTest_sqlite+pysqlite_3_41_0::test_server_default_no_false_positives[DATETIME-(now())] (call)' : sqlite
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareServerDefaultTest_sqlite+pysqlite_3_41_0::test_server_default_no_false_positives[VARCHAR(30)-substring('name', 1, 3)] (call)' : not mssql and not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareServerDefaultTest_sqlite+pysqlite_3_41_0::test_server_default_no_false_positives[VARCHAR(30)-substring('name',1,3)] (call)' : not mssql and not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareServerDefaultTest_sqlite+pysqlite_3_41_0::test_server_default_no_false_positives[VARCHAR(30)-substring(:substring_1, :substring_2, :substring_3)] (call)' : not mssql and not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareServerDefaultTest_sqlite+pysqlite_3_41_0::test_server_default_no_false_positives[VARCHAR(50)-substring(user_name(), charindex('',user_name())+(1),len(user_name()))] (call)' : not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_diffs.py::CompareServerDefaultTest_sqlite+pysqlite_3_41_0::test_server_default_no_false_positives[VARCHAR(50)-substring(user_name(),charindex('',user_name())+(1),len(user_name()))] (call)' : not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_autogen_diffs.PGCompareMetaData' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_autogen_indexes.py::AutogenerateIndexTest_sqlite+pysqlite_3_41_0::test_nothing_changed_covering_index (call)' : not postgresql >= (11,) and not mssql
SKIPPED [4] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: unique constraint reflection disabled for this suite
SKIPPED [52] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_batch.BatchRoundTripMySQLTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [53] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_batch.BatchRoundTripPostgresqlTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_batch.py::BatchRoundTripTest_sqlite+pysqlite_3_41_0::test_add_column_auto_identity (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_batch.py::BatchRoundTripTest_sqlite+pysqlite_3_41_0::test_add_column_comment (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_batch.py::BatchRoundTripTest_sqlite+pysqlite_3_41_0::test_add_column_comment_recreate (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_batch.py::BatchRoundTripTest_sqlite+pysqlite_3_41_0::test_add_table_comment (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_batch.py::BatchRoundTripTest_sqlite+pysqlite_3_41_0::test_alter_column_comment (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_batch.py::BatchRoundTripTest_sqlite+pysqlite_3_41_0::test_alter_column_comment_recreate (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_batch.py::BatchRoundTripTest_sqlite+pysqlite_3_41_0::test_drop_table_comment (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_mssql.py::OpTest::test_add_column_identity[kw0] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_mssql.py::OpTest::test_add_column_identity[kw1] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_mssql.py::OpTest::test_add_column_identity[kw2] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_mssql.py::OpTest::test_add_column_identity[kw3] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_mssql.py::OpTest::test_alter_column_identity_add_not_supported[<lambda>-<lambda>0] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_mssql.py::OpTest::test_alter_column_identity_add_not_supported[<lambda>-<lambda>1] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_mssql.py::OpTest::test_alter_column_identity_add_not_supported[<lambda>-<lambda>2] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [2] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_mssql.RoundTripTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [6] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_mysql.MySQLBackendOpTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [9] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_mysql.MySQLDefaultCompareTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_op.py::OpTest::test_create_table_comment_op (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_op.py::OpTest::test_drop_table_comment_op (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_column_identity[kw0-None] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_column_identity[kw1-None] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_column_identity[kw2-ORDER] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_column_identity[kw3-INCREMENT BY 33 START WITH 3 MAXVALUE 99 CYCLE] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_column_identity[kw4-START WITH 42] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_column_with_comment (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_identity_to_column[kw0-None] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_identity_to_column[kw1-None] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_identity_to_column[kw2-CYCLE] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_identity_to_column[kw3-INCREMENT BY 33 START WITH 3 MAXVALUE 99 CYCLE] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_add_identity_to_column[kw4-START WITH 42] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_change_identity_in_column[existing0-updated0-None] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_change_identity_in_column[existing1-updated1-START WITH 3] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_change_identity_in_column[existing2-updated2-INCREMENT BY 1 START WITH 3 MINVALUE -3 MAXVALUE 99 CYCLE] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_change_identity_in_column[existing3-updated3-START WITH 3 CACHE 2 ORDER] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_change_identity_in_column[existing4-updated4-MINVALUE 0] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_create_table_with_column_comments (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_oracle.py::OpTest::test_remove_identity_from_column (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_postgresql.PGAutocommitBlockTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_postgresql.PGAutocommitBlockTestFuture' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [7] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_postgresql.PGUniqueIndexAutogenerateTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [26] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_postgresql.PostgresqlDefaultCompareTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [4] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_postgresql.PostgresqlDetectSerialTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_postgresql.PostgresqlInlineLiteralTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_add_column_identity[kw0-None] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_add_column_identity[kw1-None] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_add_column_identity[kw2-INCREMENT BY 33 START WITH 3 MAXVALUE 99 CYCLE] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_add_identity_to_column[kw0-None] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_add_identity_to_column[kw1-None] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_add_identity_to_column[kw2-INCREMENT BY 33 START WITH 3 MAXVALUE 99 CYCLE] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_change_identity_in_column[existing0-updated0-SET GENERATED ALWAYS] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_change_identity_in_column[existing1-updated1-SET GENERATED BY DEFAULT SET START WITH 3] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_change_identity_in_column[existing2-updated2-SET CYCLE SET INCREMENT BY 1] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_change_identity_in_column[existing3-updated3-SET CACHE 2] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_change_identity_in_column[existing4-updated4-SET MINVALUE 0] (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_postgresql.py::PostgresqlOpTest::test_remove_identity_from_column (call)' : not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [4] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_script_consumption.BranchedOnlineTransactionalDDLTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'SQLAlchemy 1.x test
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_script_consumption.LegacyApplyVersionsFunctionalTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'SQLAlchemy 1.x test
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_sqlite.py::SQLiteTest::test_add_column_with_comment_ignored (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_sqlite.py::SQLiteTest::test_create_table_with_comment_ignored (call)' : custom function
SKIPPED [2] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests.test_stubs.TestStubFiles' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'black and zimports are required for this test, python 3.11 is required
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AlterColumnTest_sqlite+pysqlite_3_41_0::test_all_existings_filled[False] (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AlterColumnTest_sqlite+pysqlite_3_41_0::test_all_existings_filled[True] (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AlterColumnTest_sqlite+pysqlite_3_41_0::test_all_existings_filled_in_comment[False] (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AlterColumnTest_sqlite+pysqlite_3_41_0::test_all_existings_filled_in_comment[True] (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AlterColumnTest_sqlite+pysqlite_3_41_0::test_all_existings_filled_in_notnull[False] (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AlterColumnTest_sqlite+pysqlite_3_41_0::test_all_existings_filled_in_notnull[True] (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AlterColumnTest_sqlite+pysqlite_3_41_0::test_all_existings_filled_in_server_default[False] (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AlterColumnTest_sqlite+pysqlite_3_41_0::test_all_existings_filled_in_server_default[True] (call)' : custom function
SKIPPED [8] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'alembic.testing.suite.test_autogen_comments.AutogenerateCommentsTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateComputedTest_sqlite+pysqlite_3_41_0::test_computed_unchanged[<lambda>-_exclusions_03] (call)' : custom function
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateComputedTest_sqlite+pysqlite_3_41_0::test_remove_computed_default_on_computed (call)' : backend reflects computed construct as a server default
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateFKOptionsTest_sqlite+pysqlite_3_41_0::test_add_deferrable (call)' : not postgresql and not oracle
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateFKOptionsTest_sqlite+pysqlite_3_41_0::test_add_initially_deferrable_nochange_one (call)' : not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateFKOptionsTest_sqlite+pysqlite_3_41_0::test_add_initially_deferrable_nochange_three (call)' : not postgresql and not oracle
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateFKOptionsTest_sqlite+pysqlite_3_41_0::test_add_initially_deferrable_nochange_two (call)' : not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateFKOptionsTest_sqlite+pysqlite_3_41_0::test_add_initially_deferred (call)' : not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateFKOptionsTest_sqlite+pysqlite_3_41_0::test_add_initially_immediate_plus_deferrable (call)' : not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateFKOptionsTest_sqlite+pysqlite_3_41_0::test_remove_deferrable (call)' : not postgresql and not oracle
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateFKOptionsTest_sqlite+pysqlite_3_41_0::test_remove_initially_deferred (call)' : not postgresql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutogenerateFKOptionsTest_sqlite+pysqlite_3_41_0::test_remove_initially_immediate_plus_deferrable (call)' : not postgresql
SKIPPED [12] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'alembic.testing.suite.test_autogen_identity.AutogenerateIdentityTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'not postgresql >= (10,) and not oracle >= (12,) and not mssql
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::AutoincrementTest_sqlite+pysqlite_3_41_0::test_alter_column_autoincrement_compositepk_explicit_true (call)' : not supported by database
SKIPPED [6] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'alembic.testing.suite.test_op.BackendAlterColumnTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'no ALTER COLUMN support
SKIPPED [5] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'alembic.testing.suite.test_autogen_fks.IncludeHooksTest' unsupported on any DB implementation 'sqlite(3.41.0)+pysqlite'sqlite
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::MigrationTransactionTest_sqlite+pysqlite_3_41_0::test_autocommit_block (call)' : not postgresql and not mysql and not mariadb
SKIPPED [1] ../../../../../usr/lib64/python3.8/site-packages/sqlalchemy/testing/config.py:373: 'tests/test_suite.py::MigrationTransactionTest_sqlite+pysqlite_3_41_0::test_autocommit_block_no_transaction (call)' : not postgresql and not mysql and not mariadb
FAILED tests/test_postgresql.py::PostgresqlAutogenRenderTest::test_inline_exclude_constraint_text
================= 1 failed, 1438 passed, 310 skipped in 12.99s =================
```
</details>

Here is list of installed modules in build env
<details>

```console
Package                       Version
----------------------------- -----------------
alabaster                     0.7.13
attrs                         22.2.0
Babel                         2.12.1
build                         0.10.0
changelog                     0.5.8
charset-normalizer            3.1.0
distro                        1.8.0
docutils                      0.19
exceptiongroup                1.0.0
gpg                           1.18.0-unknown
greenlet                      1.1.3
idna                          3.4
imagesize                     1.4.1
importlib-metadata            6.0.0
importlib-resources           5.12.0
iniconfig                     2.0.0
Jinja2                        3.1.2
libcomps                      0.1.19
Mako                          1.2.4.dev0
MarkupSafe                    2.1.2
packaging                     23.0
pip                           22.3.1
pluggy                        1.0.0
Pygments                      2.14.0
pyproject_hooks               1.0.0
pytest                        7.2.2
python-dateutil               2.8.2
pytz                          2022.4
requests                      2.28.2
rpm                           4.17.0
setuptools                    65.6.3
six                           1.16.0
snowballstemmer               2.2.0
Sphinx                        6.1.3
sphinx-copybutton             0.5.1
sphinx-paramlinks             0.5.2
sphinxcontrib-applehelp       1.0.4
sphinxcontrib-devhelp         1.0.2.dev20230202
sphinxcontrib-htmlhelp        2.0.0
sphinxcontrib-jsmath          1.0.1.dev20230128
sphinxcontrib-qthelp          1.0.3.dev20230128
sphinxcontrib-serializinghtml 1.1.5
SQLAlchemy                    2.0.4.dev0
tomli                         2.0.1
typing_extensions             4.4.0
urllib3                       1.26.12
wheel                         0.38.4
zipp                          3.15.0
```
</details>

**Comment by zzzeek** [comment]
if you can use SQLAlchemy 2.0.5 it will work

cc @CaselIT not doing too well w/ regressions on this one

**Comment by CaselIT** [comment]
we were lazy and added only v2 as requirement for the test.

I think we could improve that requirement, but it's not that critical

**Comment by zzzeek** [comment]
lazy yesterday means work for me today :(

**Comment by CaselIT** [comment]
I don't think this warrant a release.

**Comment by kloczek** [comment]
> if you can use SQLAlchemy 2.0.5 it will work

Theoretically yes I can however it would be hard to resolve that as I'm using packaged modules as rpm package it would be a bit tricky. 😛 
Just checked and 
````console
[tkloczko@pers-jacek SPECS]$ grep 'BuildRequires:.*python3dist(sqlalchemy)' * -l | grep -v python-sqlalchemy.spec
python-aiopg.spec
python-alembic.spec
python-beaker.spec
python-celery.spec
python-databases.spec
python-flask-sqlalchemy.spec
python-jupyter-cache.spec
python-kombu.spec
python-logbook.spec
python-pecan.spec
python-rdflib-sqlalchemy.spec
python-sphinxcontrib-websupport.spec
```
and `alembic` seems is only module which at the moment has some issues with latest `sqlalchemy` 😋  ,, so I'll be waiting on new release hopefully with proper latest `sqlalchemy` support 😄

**Comment by CaselIT** [comment]
Oh, so the issue is that the last version of sqlalchemy in rpm is 2.0.4 (as of today) and that breaks with that test of alembic 1.10.1. correct?

I guess an option is to wait for 2.0.5 to be released in rpm form

**Comment by kloczek** [comment]
> I guess an option is to wait for 2.0.5 to be released in rpm form

I'm using my own distribution. 😋

**Comment by zzzeek** [comment]
I'm sure I've suggested this before, but if you are a packaging maintainer, you should have the ability to apply patchfiles to software before you build packages.  in this case you want to patch out the tests that were added at https://github.com/sqlalchemy/alembic/commit/6f8d9af59aa431aca3be851066c729c52b12a7a2#L1186

**Comment by kloczek** [comment]
I no longer see in 1.13.1 failing that unit. I see other fails but I'll open for that new ticket.
Closing.
Thank you..

**Comment by kloczek** [comment]
Just FTR: 1.13.1 is failing in some units however 1.13.1 + patches from master is OK.
